### ISSUE TITLE
"This extension is no longer available because it doesn't follow best practices for Chrome extensions."

### ISSUE THREAD

**Issue by jidanni** [athyuttamre/locale-switcher#61]
"This extension is no longer available because it doesn't follow best practices for Chrome extensions." - https://chromewebstore.google.com/detail/locale-switcher/kngfjpghaokedippaapkfihdlmmlafcc

**Comment by ccelik97** [comment]
Yeah, it's a good extension, and I hope it'll be updated to support Manifest V3 (MV3) too, soon enough.

For now, though, we have the option of continuing to use MV2 kind of extensions until the very end, such as by doing the "Windows Registry" thing for Chrome, Edge, etc.

It still works on Chrome stable as of writing, here, as in.

**Comment by cupcakearmy** [comment]
I've submitted the v3 manifest, its in review rn
